javaparser-core/src/main/javacc/java.jj

```diff
@@ -2547,17 +2610,12 @@ ConstructorDeclaration ConstructorDeclaration(ModifierHolder modifier):
         ("," throwType = AnnotatedReferenceType() { throws_ = add(throws_, throwType); })*
     ]
     "{" { blockBegin=token(); }
-    [
-        LOOKAHEAD(ExplicitConstructorInvocation())
-        exConsInv = ExplicitConstructorInvocation()
-    ]
+    // JEP 513: Flexible Constructor Bodies
+    // ExplicitConstructorInvocation can now appear anywhere in the body, not just at the beginning
```

Comment:
Without the context of the lines that have been removed, this comment does not make sense. I would remove it since the same is already explained below.